### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by shashwat40**
Some more content is added to the file README.md.

### CODE REVIEW

**Comment by shashwat40** [comment]
Guys I'm committing the changes now.

**Comment by MadhavGotmare** [comment]
done!!

**Comment by 01ashwini** [comment]
ok done.  :)

File: README.md [hunk 1]
```diff
@@ -1,2 +1,3 @@
 # central-railways
```
Comment (MadhavGotmare) [inline README.md]: yeah!!

**Comment by shashwat40** [comment]
Everyone joined?

**Comment by shashwat40** [comment]
FANTASTIC GUYS!!!!!!! GREAT WORK.

**Comment by shashwat40** [comment]
OK. So that now everyone is here. I'm committing the changes.

**Comment by apid01** [comment]
yeah!!

**Comment by 01ashwini** [comment]
apurva vaibhav ayush?????

**Comment by 01ashwini** [comment]
appy is apurav???

**Pull request merged**